### PR TITLE
Finished layout for OneDayFragment.kt except the sunrise part from th…

### PR DESCRIPTION
**Pull request by SemenciucCosmin**
…e bottom of the screen. Also I realised I need data from two apis: one hourly for today and tomorrow and one daily for 10 days including the current one. The next thing to implement will be either the sunrise layout part or displaying the information.

### CODE REVIEW

File: app/src/main/java/com/example/frogweather/data/NetworkRepository.kt [hunk 1]
```diff
@@ -3,11 +3,27 @@ package com.example.frogweather.data
...
+
+    private suspend fun getDailyForecastCall(latitude: Double, longitude: Double, millis: Long): CallResult<Forecast> {
+        val response = FrogWeatherApi.dailyForecastRetrofitService.getDailyForecast(latitude, longitude, millis)
+        return if (!response.isSuccessful) {
+            CallResult.Error(Exception(response.errorBody().toString()))
+        } else {
+            if (response.body() == null) {
+                CallResult.Error(Exception("Response body cannot be null."))
+            } else {
+                CallResult.Success(response.body()!!)
```
Comment (andreiprotiuc) [inline app/src/main/java/com/example/frogweather/data/NetworkRepository.kt]: I said before, extract response.body() to variable and after that check for nullability, you will not need to use !! anymore

File: app/src/main/java/com/example/frogweather/ui/NetworkViewModel.kt [hunk 2]
```diff
@@ -15,16 +15,25 @@ class NetworkViewModel : ViewModel() {
...
             }
         }
     }
 
+    fun requestHourlyForecast(latitude: Double, longitude: Double) {
+        viewModelScope.launch {
+            _uiState.value = _uiState.value?.copy(isLoading = true)
+            when (val result = networkRepository.getHourlyForecast(latitude, longitude)) {
+                is CallResult.Success -> _uiState.value = _uiState.value?.copy(isLoading = false, forecast = result.data, errorMessage = null)
+                is CallResult.Error -> _uiState.value = _uiState.value?.copy(isLoading = false, errorMessage = "Connection not available.")
```
Comment (andreiprotiuc) [inline app/src/main/java/com/example/frogweather/ui/NetworkViewModel.kt]: Message is hardcoded

**Review by andreiprotiuc — APPROVED** [review]

File: app/src/main/java/com/example/frogweather/data/NetworkRepository.kt (hunk 1 above)
Comment (SemenciucCosmin) [reply app/src/main/java/com/example/frogweather/data/NetworkRepository.kt]: done

File: app/src/main/java/com/example/frogweather/ui/NetworkViewModel.kt (hunk 2 above)
Comment (SemenciucCosmin) [reply app/src/main/java/com/example/frogweather/ui/NetworkViewModel.kt]: done

**Pull request merged**